src/_layout/components/NavBarMobile/NavBarMobile.styled.tsx

```diff
@@ -0,0 +1,13 @@
+import { styled } from "@mui/material";
+
+export const NavBarMobileStyled = styled("nav")`
+  background: black;
+  display: flex;
+  justify-content: space-between;
+  padding: 32px 24px;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  z-index: 10;
+`;
```

Comment:
I would spend some time to verify if there is a way to not have two different components for the mobile and desktop nav bar, using media query to update the style